docs/reference/sdk/classes/core.PackDefinitionBuilder.md

```diff
@@ -385,7 +385,7 @@ pack.setUserAuthentication({
+| `authDef` | `Object` |
```

Comment:
I wonder if typedoc would make more of an effort to document the parameters to `setUserAuthentication()` here if the `AuthenticationDef` type was re-exported from `index.ts`. Or even if it doesn't, it might be useful to manually link to `AuthenticationDef` from the parameter description or something.

I don't have strong feelings about this, but if there's a simple way to continue directing people looking at this parameter to something more descriptive than "object" that could be nice.

Reply:
I introduced a new `UserAuthenticationDef` type so that there's something other than "object" here.